custom_components/chihiros/chihiros_led_control/chihirosctl.py

```diff
@@ -41,25 +41,56 @@ async def _async_func() -> None:
 
 @app.command()
 def list_devices(timeout: Annotated[int, typer.Option()] = 5) -> None:
-    table = Table("Name", "Address")
+    """List all bluetooth devices.
+
+    TODO: add an option to show only Chihiros devices
+    """
+    table = Table("Name", "Address", "Model")
     discovered_devices = asyncio.run(BleakScanner.discover(timeout=timeout))
     for device in discovered_devices:
-        table.add_row(device.name, device.address)
+        assert device.name is not None
```

Comment:
if there are devices nearby without a name this command doesn't work anymore

Reply:
maybe only print out devices with a name

Reply:
I think the name is always set (and stay XXXXMACADDRESS) even when you change it in the app

Reply:
This might be true for chihiros devices but not for every Bluetooth device